trio/_core/_tests/test_asyncgen.py

```diff
@@ -257,16 +267,16 @@ def abort_fn(_):
 
 
 @pytest.mark.skipif(buggy_pypy_asyncgens, reason="pypy 7.2.0 is buggy")
-async def test_fallback_when_no_hook_claims_it(capsys):
```

Comment:
`capsys` is [`pytest.CaptureFixture`](https://docs.pytest.org/en/stable/reference/reference.html#pytest.capture.capsys)

Reply:
Changed in c06e788

Reply:
Actually it should be `CaptureFixture[str]`, the arg determines if the results are strings or bytes.

Reply:
Ah, that makes sense, I couldn't figure it out from looking at it what the generic was trying to communicate

Reply:
Changed in 5656eaf